src/qt/res/stylesheets/dark_stylesheet.qss

```diff
@@ -362,8 +369,11 @@ QTabBar::tab:!selected:hover {
     background-color: rgb(58, 64, 69);
 }
 
+/* RPC Console */
 QTextEdit{
     background-color: rgb(35,38,41);
+    font-family: "Inconsolata";
+    font-size: 10pt;
 }
```

Comment:
Is this scoped to ANY QTextEdit field? We only want this for the RPC console I think.

Reply:
Yes your right, this needs to be constrained to the RPC console only as it does reference the scope in other places.  Working on the changes.

Reply:
This has been constrained to the RPC console only.